x/application/client/config/application_configs_reader.go

```diff
@@ -0,0 +1,37 @@
+package config
+
+import (
+	"gopkg.in/yaml.v2"
+
+	sharedhelpers "github.com/pokt-network/poktroll/x/shared/helpers"
+)
+
+// YAMLApplicationConfig is the structure describing a single service stake entry in the stake config file
+type YAMLApplicationConfig struct {
+	ServiceIds []string `yaml:"service_ids"`
+}
+
+// ParseApplicationConfig parses the stake config file and returns a slice of serviceIds
+func ParseApplicationConfigs(configContent []byte) ([]string, error) {
```

Comment:
Per my previous comment, wdyt of having this return `sharedtypes.ApplicationServiceConfig`?